### PR TITLE
Use more intuitive method names in 'AdvancedSettingsDialog'

### PR DESCRIPTION
**Pull request by labkey-tchad**

### CODE REVIEW

File: src/org/labkey/test/components/domain/AdvancedFieldSetting.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package org.labkey.test.components.domain;
+
+import org.labkey.test.components.PropertiesEditor;
+
+import java.util.function.BiConsumer;
+
+public class AdvancedFieldSetting<T>
```
Comment (labkey-chrisj) [inline src/org/labkey/test/components/domain/AdvancedFieldSetting.java:7]: This approach seems interesting; would definitely like to talk a bit about how it works

File: src/org/labkey/test/components/domain/AdvancedSettingsDialog.java [hunk 2]
```diff
@@ -28,43 +29,56 @@ public AdvancedSettingsDialog(DomainFieldRow row)
...
     }
+
     public AdvancedSettingsDialog showInDefaultView(boolean checked)
     {
         elementCache().showInDefaultView.set(checked);
         return this;
     }
 
-    public boolean showOnInsertView()
+    public boolean isShownOnInsertView()
```
Comment (labkey-chrisj) [inline src/org/labkey/test/components/domain/AdvancedSettingsDialog.java:52]: general note: the method names here are pretty intuitive; I like this

**Review by labkey-chrisj — APPROVED** [review]

File: src/org/labkey/test/components/domain/DomainFieldRow.java [hunk 3]
```diff
@@ -74,9 +74,6 @@ public Input nameInput()
 
     /**
      * selects the field data type.  Note: after the field is initially created, the select will be disabled
-     *
-     * @param columnType
-     * @return
      */
```
Comment (labkey-danield) [inline src/org/labkey/test/components/domain/DomainFieldRow.java:77]: The signature didn't change, the @param and @return values in the javadocs comment should be filled in.

File: src/org/labkey/test/components/domain/DomainFieldRow.java [hunk 4]
```diff
@@ -214,8 +217,6 @@ public boolean isNewField()
 
     /**
      * indicates that the field has been edited (but is not new). The user will need to save changes to persist
-     *
-     * @return
```
Comment (labkey-danield) [inline src/org/labkey/test/components/domain/DomainFieldRow.java:218]: Same comment as above.

**Review by labkey-danield — APPROVED** [review]

**Pull request merged**